Dockerfile_dev

```diff
@@ -0,0 +1,105 @@
+# This Dockerfile used for local development.
+# Not to be confused with what we used in production.
+# The development environment is very different from production w.r.t. tools
+# installed etc.
+# This Dockerfile installs dependencies for the web app and the pipeline.
+FROM ubuntu:22.04
+
+RUN apt-get update \
+    && apt-get -y upgrade \
+    && apt-get -y install --no-install-recommends build-essential \
```

Comment:
Compared to the previous set up instructions, there are dependencies that aren't here like: jq, pigz, openconnect, watchman, lefthook, coreutils, and grep. I'm not sure why they were added and probably some are because of deployment specific pipeline code, but I think some should be included.

Reply:
Guilty.

I always question a long list of dependencies - since there's no easy way to validate you actually need them. Was worried about copy/paste creep. I was taking the approach that I'd add them in when things break.

I've added them all in can you please peek in `Dockerfile_dev` and double check that all the dependencies are actually needed?

Reply:
@AbbyBeeler watchman is being problematic. Having trouble building it. There's no arm .deb ; it builds forever and fails. Seems to be a known issue with watchman right now....

Reply:
Figured it out - using old watchman from ubuntu, installing python watchman client, and added in a sleep to workaround timeout issues.

Reply:
Nice. It's used for `yarn translations` btw